client/driver/executor/executor_linux.go

```diff
@@ -83,7 +83,8 @@ func (e *UniversalExecutor) configureCgroups(resources *structs.Resources) error
 		// Total amount of memory allowed to consume
 		e.resConCtx.groups.Resources.Memory = int64(resources.MemoryMB * 1024 * 1024)
 		// Disable swap to avoid issues on the machine
-		e.resConCtx.groups.Resources.MemorySwap = int64(-1)
```

Comment:
Is there a reason to remove this? Is it possible that in some rare situations it may work while the swappiness approach may not?

Reply:
> Is there a reason to remove this? 

Yes

1.) Writing `-1` to the `memsw.limit_in_bytes` file does not disable swap. It sets the limit to `unlimited`.
> NOTE: We can write "-1" to reset the *.limit_in_bytes(unlimited).

https://www.kernel.org/doc/Documentation/cgroup-v1/memory.txt

I'll update my commit message accordingly.

2.) It's sufficient to disable swapping via a single mechanism.

-------------------
> Is it possible that in some rare situations it may work while the swappiness approach may not?

Not that I'm aware off. The swappiness value can be set when
 `CONFIG_MEMCG` is enabled and the cggroup memory fs is mounted.
Both are also preconditions for enabling `CONFIG_MEMCG_SWAP_ENABLED`.